src/components/application_manager/rpc_plugins/rc_rpc_plugin/src/resource_allocation_manager_impl.cc

```diff
@@ -475,12 +436,9 @@ void ResourceAllocationManagerImpl::OnApplicationEvent(
     for (; acquired_modules.end() != module; ++module) {
       ReleaseResource(*module, application->app_id());
     }
-    if (!acquired_modules.empty()) {
-      SendOnRCStatusNotification();
```

Comment:
@LuxoftAKutsan I think that we don't need to delete it

Reply:
@LitvinenkoIra @yang1070 fixed 96b9724307580e5aabc8794f802d024be6f66027